### ISSUE TITLE
[ntfy] Sync state frequently changes to "Idle", hampering conversations

### ISSUE THREAD

**Issue by opusforlife2** [vector-im/element-android#6993]
### Steps to reproduce

1. Select ntfy as notifications provider.
2. Open the app a few times in between doing other stuff.

#### What did you expect?

The Sync thread should stay Running.

#### What happened instead?

The Sync thread frequently says "Idle" instead of "Running(afterPause=false)". This leads to push notifications being missed until the problem is dealt with.

The fix is to swipe away the app from Recents. Then the state changes back to Running, and the missing push notifications arrive all at once.

### Application version and app store

1.4.32 F-Droid

### Will you send logs?

No

### Are you willing to provide a PR?

No

**Comment by timokoesters** [comment]
This looks like a duplicate of #7876

**Comment by Tycho-S** [comment]
> This looks like a duplicate of #7876

Yeah I'm having the exact same behaviour on Google push (FCM) so I don't think it's ntfy-specific.
